layouts/partials/disqus.html

```diff
@@ -1,7 +1,7 @@
 {{ if (.Params.comments) | or (and (or (not (isset .Params "comments")) (eq .Params.comments nil)) (.Site.Params.comments)) }}
-  {{ if .Site.DisqusShortname }}
+  {{ if .Site.Config.Services.Disqus.Shortname }}
```

Comment:
Should we error if `Site.DisqusShortname` is set? Similar to what I added in #481? Otherwise no one will know this was changed, it will just vanish from sites.

Reply:
Yeah I guess you can add that. Sounds good.

Reply:
If you'd like to, go ahead. Pretty distracted at the moment with catch up in various projects.

Reply:
My idea is to use this in the `baseof.html`
```html
{{ if le hugo.Version "0.120.0" }}
	{{ errorf "Please use Hugo version 0.120.0 or higher!"}}
{{ end }}

{{ if not (eq .Site.DisqusShortname "") }}
	{{ errorf "Your use of disqusShortname is deprecated by Hugo; Please reference the README.md for further instructions.
	This error message will be removed in the future because we are also using .Site.DisqusShortname for checking this."}}
{{ end }}
 ```

Reply:
Seems reasonable. I can move the other check (Site.Author) there too after it's in.